src/cache.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * This module defines a cache() function for caching the strings returned by
+ * other functions.
+ *
+ * KumaScript macros originally used memcached for caching. This
+ * module is a new backend to replace memcached. Currently it just use
+ * a LRU cache in local instance memory, which means that if we have
```

Comment:
Nit: "Currently it just *uses an* LRU cache"?

Reply:
Fixed in the cache.js rewrite to use Redis.